lms/djangoapps/teams/views.py

```diff
@@ -1183,12 +1182,12 @@ class MembershipListView(ExpandableFieldViewMixin, GenericAPIView):
 
             * username: Returns membership records only for the specified user.
               If the requesting user is not staff then only memberships for
-              teams associated with courses in which the requesting user is
+              teams associated with teamsets in which the requesting user is
```

Comment:
Not sure these 2 udpates are accurate.

Reply:
Yeah, this change doesn't really make sense. I'd put it back the way it was.

Reply:
Put it back the way it was.